src/cdk-experimental/combobox/combobox-panel.ts

```diff
@@ -44,6 +44,10 @@ export class CdkComboboxPanel<T = unknown> {
 
   /** Registers the content's id and the content type with the panel. */
   _registerContent(contentId: string, contentType: AriaHasPopupValue) {
+    if (this.contentType && this.contentType !== contentType) {
+      return;
+    }
```

Comment:
Add a comment that explains that changing the content type after opening isn't allowed?